public/src/components/dashboard/SubmissionsTable.tsx

```diff
@@ -57,7 +57,7 @@ const SubmissionsTable = (): JSX.Element => {
                                 <span className="badge ml-2 float-right"><i className="fa fa-users" title="Group Assignment" /></span> : null}
                         </td>
                         <td><ProgressBar assignmentIndex={assignment.order - 1} courseID={courseID.toString()} submission={submission} type={Progress.OVERVIEW} /></td>
-                        <td>{getFormattedTime(assignment.deadline)}</td>
+                        <td>{assignment.deadline}</td>
```

Comment:
This is interesting. Don't we need to format the timestamp here? If the default formatter is nice, then maybe we don't need the `getFormattedTime()`; hehe...

Reply:
That said since we also use the `Deadline` (interface type) for other formatted stuff, perhaps it would also make sense to add the actual deadline to the `Deadline` type, as `deadline.deadline`. (Not sure it is needed though, if it already works correctly as it is.)

Reply:
> This is interesting. Don't we need to format the timestamp here? If the default formatter is nice, then maybe we don't need the `getFormattedTime()`; hehe...

Yes, my bad, we do need to format it here as well.